index.html

```diff
@@ -400,6 +400,15 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
                   				<span>Member, Software Team</span>                  
                   			</div>
                   		</div>
+                </article>
+                	<article class="col-md-2 isotopeItem software">
```

Comment:
The indentation isn't really accurate. But the whole file's indentation is a mess so I guess it is okay.